src/components/Verse/VerseSettings.tsx

```diff
@@ -0,0 +1,23 @@
+import Verse from 'types/Verse';
+import styled from 'styled-components';
+import Dropdown from '../dls/Dropdown/Dropdown';
+import MenuMoreIcon from '../../../public/icons/menu_more_horiz.svg';
+import VerseSettingsMenu from './VerseSettingsMenu';
+
+interface Props {
+  verse: Verse;
+}
+
+const VerseSettings: React.FC<Props> = ({ verse }) => (
+  <Dropdown overlay={<VerseSettingsMenu verse={verse} />}>
+    <IconContainer>
+      <MenuMoreIcon />
```

Comment:
1) Let's call the 3 dots `OverflowMenu` instead of `MenuMore`

2) We should not use "Icons" here. We should use the higher-level component definitions instead. E.g. IconContainer -> Container. MenuMoreIcon -> OverflowMenu. Inside the overflow Menu, you're going to define the icon usage. Lmk if it makes sense!

Reply:
We used `copy-to-clipboard` for writing text to clipboard in [v2](https://github.com/quran/quran.com-frontend-v2/blob/c33675c1994cecf8041134ee2209fe1dbcf2badc/app/javascript/controllers/verse_controller.js#L74)

Reply:
Okay got it.